test/components/views/settings/tabs/user/PreferencesUserSettingsTab-test.tsx

```diff
@@ -0,0 +1,122 @@
+
+/*
+Copyright 2022 Šimon Brandner <simon.bra.ag@gmail.com>
```

Comment:
```suggestion
/*
Copyright 2022 Šimon Brandner <simon.bra.ag@gmail.com>
```

also not sure this is the right copyright

Reply:
@SimonBrandner fix still has an extra blank line on it